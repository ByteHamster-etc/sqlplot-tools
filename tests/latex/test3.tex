line1
% IMPORT-DATA -T test2 test2.data
line2
% TABULAR SELECT LOG(2,testsize) AS x, bandwidth, rate, areasize FROM test2 ORDER BY x LIMIT 10
line3
indentation start
  % TABULAR SELECT LOG(2,testsize) AS x, bandwidth, rate, areasize FROM test2 ORDER BY x
  10.1699250014423124 & 21256634665.4957 &  3.7635308344391e-10 & 1024 \\ line1
  10.1699250014423124 & 20004857425.2276 &   3.999028750843e-10 & 1024 \\[40pt]
  11.0874628412503394 & 20059162540.3994 & 3.98820239074683e-10 & 2048 \\[1pt]
  11.0874628412503394 & 21206797610.8078 &  3.7723753236193e-10 & 2048 \\ line4
  11.6438561897747247 & 20101648154.6247 & 3.97977317007187e-10 & 3072 \\ line5
  11.6438561897747247 &  21256759869.173 & 3.76350866700139e-10 & 3072 \\ line6
  12.0443941193584534 & 20078735758.2602 &  3.9843145984472e-10 & 4096 \\
  12.0443941193584534 & 21256469275.0109 & 3.76356011739203e-10 & 4096 \\
  12.6147098441152082 & 21254940419.1323 & 3.76383082814898e-10 & 6144 \\
  12.6147098441152082 & 20103335011.5598 & 3.97943923005803e-10 & 6144 \\
  % END TABULAR SELECT LOG(2,testsize) AS x, bandwidth, rate, areasize FROM test2 ORDER BY x ...
indentation end
line4