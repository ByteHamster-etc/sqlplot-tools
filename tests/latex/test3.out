line1
% IMPORT-DATA -T test2 test2.data
line2
% TABULAR SELECT LOG(2,testsize) AS x, bandwidth, rate, areasize FROM test2 ORDER BY x LIMIT 10
10.1699250014423124 & 21256634665.4957 &  3.7635308344391e-10 & 1024 \\
10.1699250014423124 & 20004857425.2276 &   3.999028750843e-10 & 1024 \\
11.0874628412503394 & 20059162540.3994 & 3.98820239074683e-10 & 2048 \\
11.0874628412503394 & 21206797610.8078 &  3.7723753236193e-10 & 2048 \\
11.6438561897747247 & 20101648154.6247 & 3.97977317007187e-10 & 3072 \\
11.6438561897747247 &  21256759869.173 & 3.76350866700139e-10 & 3072 \\
12.0443941193584534 & 20078735758.2602 &  3.9843145984472e-10 & 4096 \\
12.0443941193584534 & 21256469275.0109 & 3.76356011739203e-10 & 4096 \\
12.6147098441152082 & 21254940419.1323 & 3.76383082814898e-10 & 6144 \\
12.6147098441152082 & 20103335011.5598 & 3.97943923005803e-10 & 6144 \\
% END TABULAR SELECT LOG(2,testsize) AS x, bandwidth, rate, areasize FROM tes...
line3
indentation start
  % TABULAR SELECT LOG(2,testsize) AS x, bandwidth, rate, areasize FROM test2 ORDER BY x
  10.1699250014423124 & 21256634665.4957 &  3.7635308344391e-10 &        1024 \\ line1
  10.1699250014423124 & 20004857425.2276 &   3.999028750843e-10 &        1024 \\[40pt]
  11.0874628412503394 & 21206797610.8078 &  3.7723753236193e-10 &        2048 \\[1pt]
  11.0874628412503394 & 20059162540.3994 & 3.98820239074683e-10 &        2048 \\ line4
  11.6438561897747247 & 20101648154.6247 & 3.97977317007187e-10 &        3072 \\ line5
  11.6438561897747247 &  21256759869.173 & 3.76350866700139e-10 &        3072 \\ line6
  12.0443941193584534 & 20078735758.2602 &  3.9843145984472e-10 &        4096 \\
  12.0443941193584534 & 21256469275.0109 & 3.76356011739203e-10 &        4096 \\
  12.6147098441152082 & 20103335011.5598 & 3.97943923005803e-10 &        6144 \\
  12.6147098441152082 & 21254940419.1323 & 3.76383082814898e-10 &        6144 \\
  13.0223678130284545 & 19844289277.2422 &   4.031386505323e-10 &        8192 \\
  13.0223678130284545 & 21033549756.6485 & 3.80344739359616e-10 &        8192 \\
  13.5999128421871277 & 19934071922.0195 &  4.0132292244632e-10 &       12288 \\
  13.5999128421871277 & 21105216385.1656 & 3.79053209121468e-10 &       12288 \\
  14.0112272554232541 & 21142325436.9814 &  3.7838789417207e-10 &       16384 \\
  14.0112272554232541 & 19970353548.0849 & 4.00593809255178e-10 &       16384 \\
  14.3309168781146170 & 19997700036.2933 & 4.00046004564575e-10 &       20480 \\
  14.3309168781146170 & 21164943036.4111 & 3.77983535615343e-10 &       20480 \\
  14.5924570372680804 &  20017094792.628 & 3.99658396129805e-10 &       24576 \\
  14.5924570372680804 & 21177968064.6676 & 3.77751065426661e-10 &       24576 \\
  14.8137811912170371 & 21188091979.1641 & 3.77570571614802e-10 &       28672 \\
  14.8137811912170371 & 20024361016.1429 & 3.99513372414266e-10 &       28672 \\
  15.0056245491938781 & 20167090530.7208 & 3.96685877311529e-10 &       32768 \\
  15.0056245491938781 & 19588722596.4003 & 4.08398248565229e-10 &       32768 \\
  15.3264294871223031 & 13352542574.6502 & 5.99136827707104e-10 &       40960 \\
  15.3264294871223031 & 15434595485.8038 & 5.18316142937346e-10 &       40960 \\
  15.5887146355822637 & 13352656249.9688 & 5.99131727068813e-10 &       49152 \\
  15.5887146355822637 & 15445744156.6753 & 5.17942024602459e-10 &       49152 \\
  16.0028150156070540 & 13353218261.4253 & 5.99106510758559e-10 &       65536 \\
  16.0028150156070540 & 15470586592.3006 & 5.17110321077382e-10 &       65536 \\
  16.5868397879618265 & 13353616903.6735 & 5.99088625778924e-10 &       98304 \\
  16.5868397879618265 & 15476290580.9534 & 5.16919733327156e-10 &       98304 \\
  17.0014081943928084 & 15478200720.2804 & 5.16855941111937e-10 &      131072 \\
  17.0014081943928084 &  13353488133.708 & 5.99094402892808e-10 &      131072 \\
  17.5859014496907713 & 13353703915.8915 & 5.99084722140624e-10 &      196608 \\
  17.5859014496907713 & 15452726013.7487 & 5.17708007822191e-10 &      196608 \\
  18.0007042690112466 & 15403930682.9299 & 5.19347961547587e-10 &      262144 \\
  18.0007042690112466 & 13353783611.6266 & 5.99081146787097e-10 &      262144 \\
  18.5854320515929623 &  13354375997.764 & 5.99054572174658e-10 &      393216 \\
  18.5854320515929623 & 15424185515.1548 & 5.18665960814573e-10 &      393216 \\
  19.0003521774803010 & 13355357557.2646 &   5.990105443226e-10 &      524288 \\
  19.0003521774803010 & 15423989946.1979 & 5.18672537255645e-10 &      524288 \\
  19.5851972952600240 & 15426535975.6293 & 5.18586934399162e-10 &      786432 \\
  19.5851972952600240 & 13357098956.3436 & 5.98932449789228e-10 &      786432 \\
  20.0001760994864425 & 15416610336.7218 &  5.1892081496957e-10 &     1048576 \\
  20.0001760994864425 & 13322300787.0644 & 6.00496875717426e-10 &     1048576 \\
  20.3220689761960910 &   12599911132.91 & 6.34925113011678e-10 &     1310720 \\
  20.3220689761960910 & 15270216963.0944 & 5.23895634183502e-10 &     1310720 \\
  20.5850799027671085 & 15269613288.0698 &  5.2391634608392e-10 &     1572864 \\
  20.5850799027671085 & 12597385271.8788 & 6.35052419795277e-10 &     1572864 \\
  20.8074555529676222 & 15269091481.3878 & 5.23934250426856e-10 &     1835008 \\
  20.8074555529676222 & 12593497036.6629 & 6.35248491877191e-10 &     1835008 \\
  21.0000880524301222 & 11814282322.7844 & 6.77146506357952e-10 &     2097152 \\
  21.0000880524301222 & 14380475033.7885 & 5.56309856329718e-10 &     2097152 \\
  21.1700032705344726 & 11732158553.7429 & 6.81886454513331e-10 &     2359296 \\
  21.1700032705344726 & 14310057139.6444 &  5.5904738338444e-10 &     2359296 \\
  21.3219985372613852 & 11151421728.7912 & 7.17397314402098e-10 &     2621440 \\
  21.3219985372613852 & 13667203078.2474 &  5.8534287916836e-10 &     2621440 \\
  21.4594956573012618 & 11857582908.2681 & 6.74673756185314e-10 &     2883584 \\
  21.4594956573012618 & 9635555389.30854 & 8.30258316907884e-10 &     2883584 \\
  21.5850212029383591 & 8830491065.91348 & 9.05951881983183e-10 &     3145728 \\
  21.5850212029383591 & 10975456216.8507 & 7.28899085553962e-10 &     3145728 \\
  22.0000440268868273 & 5703033760.53049 & 1.40276216763196e-09 &     4194304 \\
  22.0000440268868273 &  7462653658.7736 & 1.07200472724534e-09 &     4194304 \\
  22.3219633165043094 & 5239989138.61415 & 1.52672072181352e-09 &     5242880 \\
  22.3219633165043094 & 3764860746.89853 & 2.12491258981899e-09 &     5242880 \\
  22.5849918521283265 &  4381387103.6521 &  1.8259057715607e-09 &     6291456 \\
  22.5849918521283265 & 2970821306.34189 & 2.69285802647308e-09 &     6291456 \\
  22.8073800804431672 & 2625465946.58095 & 3.04707818070088e-09 &     7340032 \\
  22.8073800804431672 & 4033162958.21711 & 1.98355486323728e-09 &     7340032 \\
  23.0000220136113603 & 3847558417.15857 & 2.07924068529361e-09 &     8388608 \\
  23.0000220136113603 & 2448731494.05388 & 3.26699763507185e-09 &     8388608 \\
  23.1699445691134424 & 2352617641.58758 & 3.40046757219822e-09 &     9437184 \\
  23.1699445691134424 & 3743372332.88487 & 2.13711041504512e-09 &     9437184 \\
  23.3219457058033224 & 3666709518.15135 & 2.18179268371206e-09 &    10485760 \\
  23.3219457058033224 & 2288649712.68308 & 3.49551089258708e-09 &    10485760 \\
  23.5849771764993851 & 3581563981.47956 & 2.23366106018722e-09 &    12582912 \\
  23.5849771764993851 & 2222953585.66713 & 3.59881558102759e-09 &    12582912 \\
  23.8073675013052261 & 2195411712.63918 & 3.64396343243651e-09 &    14680064 \\
  23.8073675013052261 & 3543382742.29812 & 2.25772957137887e-09 &    14680064 \\
  24.0000110068476675 & 3522084255.67083 & 2.27138234615466e-09 &    16777216 \\
  24.0000110068476675 & 2180601210.44056 & 3.66871299607493e-09 &    16777216 \\
  24.3219369003722143 &  3507422082.5562 & 2.28087746832272e-09 &    20971520 \\
  24.3219369003722143 & 2170401128.14407 & 3.68595458980473e-09 &    20971520 \\
  24.5849698386289317 & 2168353166.08944 & 3.68943589315193e-09 &    25165824 \\
  24.5849698386289317 & 3503128913.41201 & 2.28367273878257e-09 &    25165824 \\
  24.8073612116951253 & 3502572094.86426 & 2.28403578379734e-09 &    29360128 \\
  24.8073612116951253 & 2167406475.24193 & 3.69104738376636e-09 &    29360128 \\
  25.0000055034343306 & 3501908751.45112 & 2.28446843358924e-09 &    33554432 \\
  25.0000055034343306 & 2167236950.08917 & 3.69133610409827e-09 &    33554432 \\
  26.0000027517197896 & 3502477870.75888 & 2.28409722921865e-09 &    67108864 \\
  26.0000027517197896 & 2167069057.70639 & 3.69162208816138e-09 &    67108864 \\
  27.0000013758605508 & 3502216012.11693 & 2.28426800983197e-09 &   134217728 \\
  27.0000013758605508 & 2167397762.24908 & 3.69106222186854e-09 &   134217728 \\
  28.0000006879304394 & 3502080053.79103 & 2.28435669005908e-09 &   268435456 \\
  28.0000006879304394 & 2167170987.53228 & 3.69144845793154e-09 &   268435456 \\
  29.0000003439652607 & 3502124096.05625 & 2.28432796228118e-09 &   536870912 \\
  29.0000003439652607 & 2167207622.93776 & 3.69138605610643e-09 &   536870912 \\
  30.0000001719826406 & 3501543006.02255 & 2.28470705235956e-09 &  1073741824 \\
  30.0000001719826406 & 2167288244.29698 & 3.69124873954872e-09 &  1073741824 \\
  31.0000000859913229 &  2167167763.2072 & 3.69145395009049e-09 &  2147483648 \\
  31.0000000859913229 & 3501652699.61996 & 2.28463548108819e-09 &  2147483648 \\
  32.0000000429956621 & 3501536124.01131 & 2.28471154278292e-09 &  4294967296 \\
  32.0000000429956621 & 2167332378.84685 & 3.69117357267392e-09 &  4294967296 \\
  33.0000000214978312 & 2167234766.24507 & 3.69133982372419e-09 &  8589934592 \\
  33.0000000214978312 & 3501106948.86093 & 2.28499160889751e-09 &  8589934592 \\
  34.0000000107489156 & 2166899399.72678 & 3.69191112471981e-09 & 17179869184 \\
  34.0000000107489156 &  3500085763.9439 & 2.28565827798048e-09 & 17179869184 \\
  % END TABULAR SELECT LOG(2,testsize) AS x, bandwidth, rate, areasize FROM tes...
indentation end
line4
